### PR TITLE
Global text corrections collected over time

### LINKED ISSUE #4: Collected corrections from DNNCommunity site
Incorporate the collected improvements found in the DNNCommunity version of the nl-NL LP

### PR DESCRIPTION
**Pull request by EPTamminga**
Close #4 4

### CODE REVIEW

File: Resources/App_GlobalResources/List_BannedPasswords-0.nl-NL.resx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (mitchelsellers) [inline Resources/App_GlobalResources/List_BannedPasswords-0.nl-NL.resx:1]: This appears to be a "Portal" specific file, I don't think this should be as part of the language pack

File: Resources/App_GlobalResources/List_ProfanityFilter-0.nl-NL.resx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (mitchelsellers) [inline Resources/App_GlobalResources/List_ProfanityFilter-0.nl-NL.resx:1]: This appears to be a "Portal" specific file, I don't think this should be as part of the language pack

Comment (EPTamminga) [reply Resources/App_GlobalResources/List_ProfanityFilter-0.nl-NL.resx]: I did some more testing. The -0 .resx files are not in the distribution LP, but are created as soon as you have installed DNN and start the site. I will remove them from the PR

File: Resources/App_GlobalResources/List_BannedPasswords-0.nl-NL.resx (hunk 1 above)
Comment (EPTamminga) [reply Resources/App_GlobalResources/List_BannedPasswords-0.nl-NL.resx]: I did some more testing. The -0 .resx files are not in the distribution LP, but are created as soon as you have installed DNN and start the site. I will remove them from the PR

**Comment by EPTamminga** [comment]
Merged with gitignore update #6

**Pull request closed without merging**